samza-core/src/main/scala/org/apache/samza/container/TaskInstance.scala

```diff
@@ -101,6 +91,16 @@ class TaskInstance(
     }
   }
 
+  def startTableManager {
+    if (tableManager != null) {
```

Comment:
Is there any case tableManager would be null for a task? It seems that it will always be not null.

Reply:
Correct as of now, but won't hurt to be defensive.